manifests/x/xushengfeng/eSearch/1.12.0/xushengfeng.eSearch.locale.en-US.yaml

```diff
@@ -0,0 +1,102 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: xushengfeng.eSearch
+PackageVersion: 1.12.0
+PackageLocale: en-US
+Publisher: xsf
+PublisherUrl: https://github.com/xushengfeng
+PublisherSupportUrl: https://github.com/xushengfeng
+Author: xushengfeng
+PackageName: eSearch
+PackageUrl: https://github.com/xushengfeng/eSearch
+License: GPL-3.0
+LicenseUrl: https://github.com/xushengfeng/eSearch/blob/HEAD/LICENSE
+ShortDescription: Screenshot OCR search translate search for picture paste the picture on the screen screen recorder
+Tags:
+- ocr
+- screen-recorder
+- screenshot
+- search
+- translate
+ReleaseNotes: |-

```

Comment:
Hi @xushengfeng,

In the future, could the release notes be made in the localization file's language?